.github/workflows/serverless.yaml

```diff
@@ -30,6 +30,10 @@ jobs:
         run: cargo check --verbose
       - name: Run cloud example
         run: cargo run --example cloud -- $HOME/.ccm/serverless/config_data.yaml
+      - name: Run session tests
+        run: CLOUD_CONFIG_PATH=$HOME/.ccm/serverless/config_data.yaml cargo test --features cloud --package scylla --lib transport::session_test
+      - name: Run cloud tests
+        run: CLOUD_CONFIG_PATH=$HOME/.ccm/serverless/config_data.yaml cargo test --features cloud --package scylla --lib cloud
```

Comment:
I don't like this approach. Now, developers have to remember to add more serverless tests if they appear. I think we should have a single `cargo test` invocation which passes `--cfg serverless-tests` and everything works correctly. It's much more reasonable to require developers to use `#[cfg_attr]` annotations and `create_new_session_builder` in tests IMO.